gpflow/training/scipy_optimizer.py

```diff
@@ -70,6 +69,9 @@ def minimize(self, model, session=None, var_list=None, feed_dict=None,
             initialized before for gotten session.
         :param anchor: If `True` trained parameters computed during optimization at
             particular session will be synchronized with internal parameter values.
+        :param step_callback: A function to be called at each optimization step;
```

Comment:
@ahsimb Same here about signature of callback, are these values tensors or numpy arrays? ;)